Makefile

```diff
@@ -194,8 +200,8 @@ build/libcxx.BUILT: build/llvm.BUILT build/compiler-rt.BUILT build/wasi-libc.BUI
 	mkdir -p build/libcxx
 	cd build/libcxx && cmake -G Ninja $(LIBCXX_CMAKE_FLAGS:@PTHREAD@=OFF) \
 		-DCMAKE_SYSROOT=$(BUILD_PREFIX)/share/wasi-sysroot \
-		-DCMAKE_C_FLAGS="$(DEBUG_PREFIX_MAP) $(EXTRA_CFLAGS)" \
-		-DCMAKE_CXX_FLAGS="$(DEBUG_PREFIX_MAP) $(EXTRA_CXXFLAGS)" \
+		-DCMAKE_C_FLAGS="$(DEBUG_PREFIX_MAP) $(EXTRA_CFLAGS) -fPIC" \
```

Comment:
If we are going to apply `-fPIC` uniformly for the time being, a more idiomatic way to do this shall be setting `CMAKE_POSITION_INDEPENDENT_CODE` to `On` in the `wasi-sdk.cmake` and `wasi-sdk-pthread.cmake` files. The flags in `wasi-sdk.cmake` will be used by `compiler-rt` and `libcxx`/`libcxxabi` builds anyway.

Reply:
Done, thanks!